backend/Controllers/SessionPlayersController.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using IP5.Model;
+using IP5.Repositories;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+
+namespace IP5.Controllers
+{
+	[ApiController]
+	[Route("[controller]")]
+	public class SessionPlayersController : ControllerBase
```

Comment:
do you need this controller? 
I would instead from the session controller manage the sessionplayers as well.

Reply:
I will refactor that soon